packages/backend/src/workers/provider/WhisperCpp.ts

```diff
@@ -0,0 +1,109 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import { InferenceProvider } from './InferenceProvider';
+import type { TaskRegistry } from '../../registries/TaskRegistry';
+import type { InferenceServer } from '@shared/src/models/IInference';
+import { InferenceType } from '@shared/src/models/IInference';
+import type { InferenceServerConfig } from '@shared/src/models/InferenceServerConfig';
+import { LABEL_INFERENCE_SERVER } from '../../utils/inferenceUtils';
+import type { MountConfig } from '@podman-desktop/api';
+import { DISABLE_SELINUX_LABEL_SECURITY_OPTION } from '../../utils/utils';
+
+export const WHISPER_CPP_CPU = 'ghcr.io/containers/whispercpp:latest';
+
+export class WhisperCpp extends InferenceProvider {
+  constructor(taskRegistry: TaskRegistry) {
+    super(taskRegistry, InferenceType.WHISPER_CPP, 'LLama-cpp');
```

Comment:
```suggestion
    super(taskRegistry, InferenceType.WHISPER_CPP, 'Whisper-cpp');
```

Reply:
Oh my god ! Nice catch I completely forgot about this string